src/contexts/W3iContext/index.tsx

```diff
@@ -50,6 +50,26 @@ const W3iContextProvider: React.FC<W3iContextProviderProps> = ({ children }) =>
     wagmiDisconnect()
   }, [wagmiDisconnect])
 
+  const pushEnabledQuery = query.get('pushEnabled')
+  const chatEnabledQuery = query.get('chatEnabled')
+  const settingsEnabledQuery = query.get('settingsEnabled')
+
+  const [uiEnabled] = useState({
+    push: pushEnabledQuery ? JSON.parse(pushEnabledQuery) : true,
+    settings: settingsEnabledQuery ? JSON.parse(settingsEnabledQuery) : true,
+    chat: chatEnabledQuery ? JSON.parse(chatEnabledQuery) : true,
+    sidebar: false
```

Comment:
Why is sidebar set to false by default ?

Reply:
Looks like the state is being re-evaluated below based on what was enabled:
https://github.com/WalletConnect/web3inbox/pull/102/files#diff-8837a0bdc83cb8d1f029e0555ca8879e97da5fbd68f45435f58680c167d7b3f2R70